web/styles/recent_view.css

```diff
@@ -44,7 +44,12 @@
         display: flex;
         align-items: center;
 
-        & > * {
+        & > .pm-row-heading,
+        & > .stream-name,
+        & > .pm-url,
+        & > .topic-url,
+        & > .zulip-icon,
+        & > .unread_count {
```

Comment:
Maybe we can use some common class here.

Reply:
You can look at the approach @sayamsamal has taken with classnames in `web/templates/popovers/` for examples of a nicer style -- each component that we want to style has class names like `popover-menu-icon` or `popover-menu-separator` that are unique across the app and don't require CSS cascading to evaluate; it's both more readable and performant.